Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_projekt1_inżynieria_cech.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>Ważne! To co nazywamy potocznie normalizacją, funkcja normalize w sklearn to dwa różne światy. Normalize działa na wierszach, nie na kolumnach! W tym przypadku MinMaxScaler albo StandardScaler byłby ok.</p><p> </p><p>To po co ten normalize? Na przykład jeśli chcemy liczyć odległość cosinusową. Nie interesuje nas wtedy długość wektora, tylko jego kierunek</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>